doc/api/timers.markdown

```diff
@@ -7,15 +7,15 @@ this module in order to use them.
 
 ## clearImmediate(immediateObject)
 
-Stops an immediate from triggering.
+Stops an `immediateObject`, as created by [`setImmediate`][], from triggering.
 
 ## clearInterval(intervalObject)
 
-Stops an interval from triggering.
+Stops an `intervalObject`, as created by [`setInterval`][], from triggering.
```

Comment:
Is that worth documenting? I don't think I want to encourage folks to clear a timeout with `clearInterval` or vice-versa.